requirements.txt

```diff
@@ -1,2 +1,4 @@
 click>=6
 requests>=2.20.0,<3
+configparser>=6
+pandas~=2.0.0
```

Comment:
Agreed - will remove on next push.

Reply:
I decided to remove the tabular output feature throughout.  Nobody asked for it (except me) and it will not be missed.  Also, dumping Prism JSON to CSV would be a pain.